src/deadify.2024t1.js

```diff
@@ -4,16 +4,16 @@
 		// see https://www.epochconverter.com/weeks/2022 (or current year) to help you here
 		// dates from https://www.wgtn.ac.nz/students/study/dates
 		let weekNum = getWeekOfYear(new Date());
-		const startWeek = 9;
+		const startWeek = 1;
```

Comment:
No, this needs to be the week of the year that tri 2 has started. 1 would mean we think tri 2 started in the first week of January.